test/integration/features/addons/addon-che-with-anyuid.feature

```diff
@@ -0,0 +1,72 @@
+@addon-che-with-anyuid @addon @core
+Feature: Che add-on in combination with anyuid addon.
+  Che addon starts Eclipse Che with applied anyuid addon.
+
+  Scenario: Che and anyuid add-ons are part of Minishift
+     When executing "minishift addons list" succeeds
+     Then stdout should contain "che"
+      And stdout should contain "anyuid"
+  
+  Scenario: User starts Minishift
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start --memory=5GB" succeeds
+     Then Minishift should have state "Running"
+
+  Scenario: User applies anyuid add-on
+     When executing "minishift addons apply anyuid" succeeds
```

Comment:
Why not enabling `anyuid` before doing the minishift start? I think that is a better way of doing it as in CDK `anyuid` is enabled bydefault.

Reply:
@LalatenduMohanty Good point. Will change that.